nss-crystal-cobras/WorkforceManagement
# Computer jd#8

# Description
**Computer**  you should be able to Create, View, and Delete.

Closes #8 (https://github.com/nss-crystal-cobras/WorkforceManagement/issues/8) 



## Type of change


# Testing Instructions

git fetch --all
git checkout ComputerJD#8
At this point, you will want to check and see if Computers View, Create, and Delete work properly.

# Checklist:
